src/NuGet.Clients/PackageManagement.VisualStudio/SolutionRestore/SolutionRestoreWorker.cs

```diff
@@ -54,6 +60,27 @@ internal sealed class SolutionRestoreWorker : ISolutionRestoreWorker, IDisposabl
 
             _componentModel = _serviceProvider.GetService<SComponentModel, IComponentModel>();
 
+            _solutionManager = _componentModel.GetService<IVsSolutionManager>();
```

Comment:
Should this new code be conditional on the VS15 compiler flag? It looks like it won't be used for VS14. It would just help reduce noise there.